src/lib/input/input-container.ts

```diff
@@ -379,4 +381,8 @@ export class MdInputContainer implements AfterContentInit {
 
     this._mdInputChild.ariaDescribedby = ids.join(' ');
   }
+
+  getPopupConnectionElementRef(): ElementRef {
```

Comment:
It feels like this should be named for the underline, given that there may be other non-popup components that might want to hook in here.